rt/bash/runsort

```diff
@@ -6,8 +6,8 @@ cd $(dirname $0)    # set to directory of script - rt/bash
 cd ..               # up to rt
 cd ..               # up to z390
 
-# rebuild sort/SORT, sort/TESTSRT[1-4]
-bash/asml sort/SORT $1 $2 $3 $4 $5
+# copy sort loadmod, assemble test programs
+cp linklib/SORT.390 sort/SORT.390
```

Comment:
Okay, but not what I would do. Sending you via email an alternative version of rt/bash/runsort.